### PR TITLE
chore: modify test script

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* add `check-types`, `lint`, `validate` npm script
* change `awesome-typescript-loader` to `ts-loader`
* when run each cypress test, type checking is not executed to improve test speed and save the memory. Instead, when start the test or build, type checking and lint are executed once in the first.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by js87zz** [comment]
ok to test

File: packages/toast-ui.grid/package.json [hunk 1]
```diff
@@ -4,15 +4,18 @@
   "description": "TOAST UI Grid : Powerful data grid control supported by TOAST UI",
   "main": "dist/tui-grid.js",
   "scripts": {
-    "start": "webpack-dev-server",
-    "build": "webpack --mode=production & webpack --mode=production --minify && node tsBannerGenerator.ts",
+    "lint": "eslint . --ext .ts,.tsx",
+    "check-types": "tsc --project ./cypress/tsconfig.json",
+    "validate": "npm run check-types && npm run lint",
+    "start": "npm run validate && webpack-dev-server",
+    "build": "npm run validate && (webpack --mode=production & webpack --mode=production --minify) && node tsBannerGenerator.ts",
```
Comment (jung-han) [inline packages/toast-ui.grid/package.json]: script 대신 웹팩 설정에 프로덕션으로 빌드할 떄 eslint-loader `failOnError`를 사용해도 괜찮지 않을까요? 이후에 빌드에 필요한 여러가지 조건들이 들어가면 이 스크립트가 점점 더 길어질 것 같습니다ㅠㅠ

Comment (js87zz) [reply packages/toast-ui.grid/package.json]: 추후 액션 적용을 위해 단계 별로 나눠놓았는데, 트래킹하기에는 이게 더 좋지 않을까요? 나중에는 저 validate라는 스크립트도 없어질거라서요.

Comment (jung-han) [reply packages/toast-ui.grid/package.json]: 넵넵 이대로 가도 좋을 것 같습니다.

**Pull request merged**